src/code/z_kaleido_scope_call.c

```diff
@@ -1,14 +1,15 @@
-#include "global.h"
+#include "z64pause_menu.h"
+
+#include "z64.h"
```

Comment:
any reason to include both when z64.h includes z64pause_menu.h ?
(this is more of a question on how you plan on splitting headers than a request to remove either include)

Reply:
Personally I like the idea that a file from a specific system should include the header of said system at the top of the file, even if it is a bit redundant because of other includes. Similar to actors including their own header at the top of the file

So kaleido files should include the `z64pause_menu.h` header explicitly because the are all part of the same system.

About how to split headers: My idea is to split them by system whenever possible and as much possible.
When we have the headers more or less splited I would like if we had headers that had the only purpose of including other common headers. For example some kind of "actor header set" that would include files that most actors usually use (`z64actor.h`, collision stuff, skel/animation stuff, `z64play.h`, etc)